utils/_context/_scenarios.py

```diff
@@ -853,6 +859,24 @@ class scenarios:
         doc="Spawns tracer, agent, and a full set of database. Test the intgrations of thoise database with tracers",
     )
 
+    otel_integrations = EndToEndScenario(
+        "OTEL_INTEGRATIONS",
+        weblog_env={
+            "DD_DBM_PROPAGATION_MODE": "full",
+            "DD_TRACE_SPAN_ATTRIBUTE_SCHEMA": "v1",
+            "OTEL_EXPORTER_OTLP_ENDPOINT": "http://agent:4317",
+        },
+        agent_env={"DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT": "0.0.0.0:4317"},
+        include_postgres_db=True,
+        include_cassandra_db=True,
+        include_mongo_db=True,
+        include_kafka=True,
+        include_rabbitmq=True,
+        include_mysql_db=True,
+        include_sqlserver=True,
+        doc="Spawns tracer, agent, and a full set of database. Test the intgrations of thoise database with tracers",
+    )
```

Comment:
We need to add env variables:

- Weblog: OTEL_EXPORTER_OTLP_ENDPOINT
- Agent: DD_OTLP_CONFIG_RECEIVER_PROTOCOLS_GRPC_ENDPOINT